application/workflows/auto_novel_generation_workflow.py

```
@@ -364,10 +380,11 @@ async def generate_chapter_stream(
             
             # 如果使用节拍模式，先放大节拍
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)